.github/workflows/security.yml

```
@@ -4,15 +4,15 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.13.x, 1.14.x, 1.15.x]
+        go-version: [1.12.x, 1.13.x, 1.14.x, 1.15.x, 1.16.x]
```

```suggestion
        go-version: [1.13.x, 1.14.x, 1.15.x, 1.16.x]
```
Seems like `gosec` doesn't work on 1.12.